### PR TITLE
Fix string encoding of creator_opts

### PR DESCRIPTION
**Pull request by gsvr**
Causing FAQ navigation issues

### CODE REVIEW

**Comment by gsvr** [comment]
@justinvdm Good work on PaginatedState - it just worked :)  Would you mind doing a review?  @imsickofmaps already reviewed on the seperate snappy-faq-browser app and gave a +1.  Would also appreciate your input on the commented translation questions in the changed files.

File: src/personal.js [hunk 1]
```diff
@@ -631,38 +635,17 @@ go.app = function() {
...
-                pages: answer_split.length,
-                page_text: function(n) {return answer_split[n];},
-                buttons: {"1": -1, "2": +1, "0": "exit"},
-                footer_text: footer_text,
+            return new PaginatedState(name, {
+                text: opts.answer,
+                exit: $("Send to me by SMS"),
+                // wrap in translation?
+                    // make sure this is going into POT files
+                    // make sure More and Back are also being translated
```
Comment (justinvdm) [inline src/personal.js]: Yup, we need to wrap `more`, `back` and `exit` in `$()`s.

File: src/personal.js [hunk 2]
```diff
@@ -631,38 +635,17 @@ go.app = function() {
...
-            }
-            answer_split.push(answer);
-
-            return new BookletState(name, {
-                pages: answer_split.length,
-                page_text: function(n) {return answer_split[n];},
-                buttons: {"1": -1, "2": +1, "0": "exit"},
-                footer_text: footer_text,
+            return new PaginatedState(name, {
+                text: opts.answer,
```
Comment (justinvdm) [inline src/personal.js:639]: We need the text to be string literals (eg, `'foo'`) and wrapped in a `$()` for jspot to be able to extract it.

Comment (justinvdm) [inline src/personal.js:639]: Unless translation is being done a different way (eg an api is providing different content based on the user's language).

**Comment by justinvdm** [comment]
Looks good, just the the translation questions. I might not have enough context to give a good review though.

**Comment by gsvr** [comment]
@justinvdm Don't worry about the context, just wanted you to have a look at our first use of this state.  It should be fixed now.

**Comment by justinvdm** [comment]
:+1:

**Pull request merged**